modules/users/client/controllers/settings/manage-social-accounts.client.controller.js

```diff
@@ -7,7 +7,9 @@ angular.module('users').controller('SocialAccountsController', ['$scope', '$http
     // Check if there are additional accounts
     $scope.hasConnectedAdditionalSocialAccounts = function (provider) {
       for (var i in $scope.user.additionalProvidersData) {
-        return true;
```

Comment:
Also, I don't know why we have `provider` there as a parameter. It's not being used. We probably should remove it.